sentinel/test/mocha/lib/message-utils.js

```diff
@@ -0,0 +1,331 @@
+const sinon = require('sinon').sandbox.create(),
+      moment = require('moment'),
+      expect = require('chai').expect,
+      should = require('chai').should(),
+      uuid = require('uuid'),
+      utils = require('../../../lib/message-utils');
+
+const MAX_GSM_LENGTH = 160;
+const MAX_UNICODE_LENGTH = 70;
+
+var generateMessage = function(length, unicode) {
+  var result = [];
+  for (var i = 0; i < length; i++) {
+    result[i] = unicode ? '☃' : 'o';
+  }
+  return result.join('');
+};
+
+describe('messageUtils', () => {
+
+  beforeEach(done => {
+    sinon.restore();
+    done();
+  });
+
+  describe('getRecipient', () => {
+    it('is undefined if no doc is passed', () => {
```

Comment:
> `it('is undefined`
It's defined, but _returns_ `undefined`, from the looks of things.

Reply:
Done.